easybuild/easyconfigs/f/FFTW3-sve/FFTW3-sve-1.0.0-ffmpi-2105.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'EB_FFTW'
+
+name = 'FFTW3-sve'
+version = '1.0.0'
+
+homepage = 'https://github.com/fujitsu/fftw3'
+description = """This is a fork of the FFTW3 for the Armv8-A 64-bit architecture (AArch64)
+ with 512-bit Scalable Vector Extension (SVE) support."""
+
+toolchain = {'name': 'ffmpi', 'version': '2105'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/fujitsu/fftw3/archive/refs/tags/']
+sources = ['sve-v%(version)s.tar.gz']
+checksums = ['b5931e352355d8d1ffeb215922f4b96de11b8585c423fceeaffbf3d5436f6f2f']
+
+preconfigopts = "touch ChangeLog && autoreconf --verbose --install --symlink --force && "
```

Comment:
Probably needs Autotools?

Reply:
yeah, but for now I would only add an `osdep`?  (this toolchain is at least for now only meant to use on top of what is already available at Fugaku)

Reply:
I suppose an `osdep` on `binutils` would also make sense (in `ffmpi`), since the `lang` external module wouldn't work without it...